indico/modules/events/registration/forms.py

```diff
@@ -407,3 +407,11 @@ def __init__(self, *args, **kwargs):
         super(ImportRegistrationsForm, self).__init__(*args, **kwargs)
         if not self.regform.moderation_enabled:
             del self.skip_moderation
+
+
+class RejectRegistrantsForm(IndicoForm):
+    rejection_reason = TextAreaField(_('Reason'), description=_("Provide a reason for rejection of registration(s)."))
```

Comment:
What about something like "You can provide a reason for the rejection here"? This avoids the ugly "(s)" pluralization.